_config.yml

```diff
@@ -1,8 +1,9 @@
 name: ark
 baseurl: /ark
 google_analytics: FILL_OUT
-latest: v0.9.0
```

Comment:
Do we want latest to point to this still? Or only let people see it if they're specifically looking for it?

Reply:
We should continue to show 0.9 as the latest stable imho

Reply:
Only sorta done. It's taking me a little while to get the conditional logic right for a better version warning statement for 0.10.0-beta so I figure it's better to get at least a partial fix up first. Will continue to noodle (conditional logic brain evidently sleepy atm, also lazy)

Note that version warning logic doesn't work, but doesn't break.